src/devtools/client/debugger/src/components/SourceOutline/SourceOutline.tsx

```diff
@@ -139,6 +140,9 @@ export function SourceOutline({
   return (
     <div className={classnames("flex h-full flex-col space-y-2")}>
       <OutlineFilter filter={filter} updateFilter={setFilter} />
+      {symbols.hasHitCounts || (
+        <div className={styles.Warning}>Outline hit counts could not be loaded for this source</div>
```

Comment:
The reason why I wrote "Outline hit counts" is that even if the hit counts for the source's outline couldn't be loaded, the source panel will still show hit counts (because it loads them in buckets, unlike the outline panel).

Reply:
Sure, that's fair.